src/components/offer/offers.tsx

```diff
@@ -1,13 +1,32 @@
-import Offer from './offer';
+import { useState } from 'react';
+import { Nullable } from 'vitest';
+import OfferItem from './offer';
+import { TOffer } from '../../types/offers';
 
 type OffersScreenProps = {
   offersCount: number;
+  offers: TOffer[];
 }
 
-function Offers ({offersCount}: OffersScreenProps): JSX.Element {
+function Offers ({offersCount, offers}: OffersScreenProps): JSX.Element {
+
+  const [activeOffer, setActiveOffer] = useState<Nullable<TOffer>>(null);
```

Comment:
Если пока не нужен activeOffer, то можно его не деструкторизировать

```suggestion
  const [, setActiveOffer] = useState<Nullable<TOffer>>(null);
```